setup.py

```diff
@@ -193,9 +193,10 @@ def run(self):
                     required_failed.append(package)
             else:
                 good_packages.append(package)
-                if isinstance(package, setupext.OptionalBackendPackage):
-                    if default_backend is None:
-                        default_backend = package.name
+                if (isinstance(package, setupext.OptionalBackendPackage)
+                    and package.runtime_check()
```

Comment:
:bike: :house:  can you add an extra 4 spaces to these two lines so the conditional and the code can be separated?